iDiff/differs/pipDiff.go

```diff
@@ -86,11 +81,30 @@ func (d PipDiffer) getPackages(path string) (map[string]utils.PackageInfo, error
 					glog.Errorf("Could not find Python package %s for corresponding metadata info", packageName)
 					continue
 				}
+				currPackage := utils.PackageInfo{Version: version, Size: size}
+				addToMap(packages, packageName, pyVersion, currPackage)
+			}
 
-				packages[packageName] = utils.PackageInfo{Version: version, Size: size}
+			// if not package, check if Python file
```

Comment:
It's because I realized I wasn't making use of the structure correctly... Now that I realized there's a corresponding .dist-info folder for either a package directory or script, I don't separately check for directories and scripts, so in your tests, in order to catch the .py scripts, you need to create a .dist-info for each script you have.